src/pages/index.tsx

```diff
@@ -0,0 +1,415 @@
+                <p>Get fastify-cli with NPM:</p>
+                <CodeBlock className="language-bash">{'npm install --global fastify-cli'}</CodeBlock>
+                <p>Then scaffold a new project with:</p>
+                Of course, Fastify can do much more than this.
+                <CodeBlock className="language-bash">{'fastify generate myproject'}</CodeBlock>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="section alternate">
+          <div className="container content">
+            <div className="columns is-centered">
+              <div className="column is-12">
+                <h1 className="title">Request/Response validation and hooks</h1>
+                <p>Of course, Fastify can do much more than this.</p>
+                <p>
+                  For example, you can easily provide input and output validation using JSON Schema and perform specific
+                  operations before the handler is executed:
+                </p>
+                <div className="flex justify-end">
+                  <span>async/await</span> <ToggleSwitch />
+                </div>
+                {isToggled ? <ReqResHooksWithAsyncAwait /> : <ReqResHooks />}
+              </div>
+            </div>
+          </div>
+        </section>
+        <section className="section alternate">
+          <div className="container content">
+            <div className="columns is-centered">
+              <div className="column is-12">
+                <h1 className="title">TypeScript Support</h1>
+                <p>
+                  Fastify is shipped with a typings file, but you may need to install
+                  <CustomHighLight text={'@types/node'} />, depending on the Node.js version you are using.
+                </p>
+                <p>The following example creates a http server.</p>
+                <p>We pass the relevant typings for our http version used.</p>
+                <p>By passing types we get correctly typed access to the underlying http objects in routes.</p>
+                <p>
+                  If using http2 we&39;d pass
+                  <CustomHighLight text={'<http2.Http2Server, http2.Http2ServerRequest, http2.Http2ServerResponse>'} />.
+                </p>
+                <p>
+                  For https pass <CustomHighLight text={'http2.Http2SecureServer'} />
+                  or <CustomHighLight text={'http.SecureServer'} /> instead of Server.
+                </p>
+                <p>
+                  This ensures within the server handler we also get <CustomHighLight text={'http.ServerResponse'} />
+                  with correct typings on <CustomHighLight text={'reply.res'} />.
+                </p>
+                <div className="flex justify-end">
+                  <span>async/await</span> <ToggleSwitch />
+                </div>
+                {isToggled ? <TypeScriptSupportWithAsyncAwait /> : <TypeScriptSupport />}
```

Comment:
we should search for some docusaurus standard here

writing a code renderer is massive and a custom solution could hurt us

Reply:
okay, let me check out their docs